cupy/testing/helper.py

```diff
@@ -1263,14 +1262,28 @@ def test_values(self):
 class AssertFunctionIsCalled:
 
     def __init__(self, mock_mod, **kwargs):
+        """A handy wrapper for unittest.mock to check if a function is called.
+
+        This class should be used as a context manager.
+
+        Args:
+            mock_mod (str): the function to be mocked.
+            is_called (int or bool): the number of times the mocked function
+                should be called. Default is ``1``.
```

Comment:
Instead of having this as both `int` and `bool` how about adding a mutually exclusive
`times_called` argument?

Reply:
Sounds good.

Reply:
you can also just replace `is_called` and fix the tests using it

Reply:
Yeah it's easier...